src/components/Footer.tsx

```diff
@@ -1,54 +1,62 @@
-import { Box, Center, HStack, Stack } from '@chakra-ui/react'
+import { Box, Center, HStack, Stack, useColorModeValue } from '@chakra-ui/react'
 import Image from 'next/image'
 
 import poweredByVercelLogo from '@app-public/powered-by-vercel.svg'
 import { REPO_URL } from '~/common'
 import FluidContainer from '~/components/FluidContainer'
 import Link from '~/components/Link'
 
-const Footer = () => (
-  <Box as="footer" bg="gray.50" flexShrink={0}>
-    <FluidContainer py={5}>
-      <Stack justify="space-between" direction={['column', 'row']} spacing={4}>
-        <HStack
-          shouldWrapChildren
+const Footer = () => {
+  const boxBgColor = useColorModeValue('gray.50', 'gray.900')
```

Comment:
So this is the magic line for switching between 2 values depending on color mode.

You only need to pass 2 values to `useColorModeValue`: the first one is the variant for light mode, and the second one is the variant for dark mode.

Then you save it in a const that will _automagically_ get the corresponding value based on the current color mode.

You can use that variable anywhere, but most of the time will be passed to a Chakra component prop as I did in the `Box` below.

You can use `useColorModeValue` as many times you need/want inside a component to generate several variables based on color mode.

Reply:
Thank you for the example! ❤️ 

I've pushed a commit that adds your advice to the `contributing.md` file.